### PR TITLE
Fix for Bitbucket connector does not write vcs_Repository #339

### PR DESCRIPTION
**Pull request by huongtn**
## Description

> Provide description here

## Type of change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#339]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import {AirbyteRecord} from 'faros-airbyte-cdk';
+import {Utils} from 'faros-feeds-sdk/lib/utils';
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts:2]: `import {Utils} from 'faros-feeds-sdk';`

**Review by tovbinm — COMMENTED** [review]
arent there any tests to modify?

File: destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts [hunk 2]
```diff
@@ -51,6 +53,22 @@ export class Repositories extends BitbucketConverter {
           organization: {uid: workspace?.uuid, source},
         },
       },
+      {
+        model: 'vcs_Repository',
+        record: {
+          name: repository.slug.toLowerCase(),
+          fullName: repository.fullName,
```
[the remark this answers is not in the document]
Comment (huongtn) [reply destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts:60]: 1. Is there repository.fullName and repository.name ?
 yes, for ex: {"type":"repository","name":"GeekHub2019","fullName":"RomanKyrnis/geekhub2019"}
what field should I used  fullName or name?

File: destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts [hunk 3]
```diff
@@ -37,17 +39,34 @@ export class Repositories extends BitbucketConverter {
...
+          description,
+          url: repository?.links?.htmlUrl,
+          organization: {uid: workspace?.uuid, source},
+        },
+      },
+      {
+        model: 'vcs_Repository',
+        record: {
+          name: repository.slug.toLowerCase(),
+          fullName: repository.name,
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts:61]: I think we actually want `repository.fullName` (assuming it's returned from the Bitbucket source/API)

File: destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (tovbinm) [reply destinations/airbyte-faros-destination/src/converters/bitbucket/repositories.ts:61]: Whatever we have defined in the original Bitbucket feed (see feeds repo)

**Review by tovbinm — APPROVED** [review]

**Comment by tovbinm** [comment]
Thank you!

**Pull request merged**